Documentation/Books/Manual/Views/ArangoSearch/DetailedOverview.md

```diff
@@ -52,87 +52,103 @@ directives.
-    cause the view not to account for them, (unlit commit), and memory usage
-    would continue to grow
-    for the case where there are a few inserts/updates, a higher value will
-    impact performance and waste disk space for each commit call without any
-    added benefits
-
-  * **consolidate** (_optional_, default: `none`)<br/>
-    a per-policy mapping of thresholds in the range `[0.0, 1.0]` to determine data
-    store segment merge candidates, if specified then only the listed policies
-    are used, keys are any of:
-
-    * **bytes** (_optional_, for default values use an empty object: `{}`)
-
-      * **segmentThreshold** (_optional_, default: `300`; to disable use: `0`)<br/>
-        apply consolidation policy IFF {segmentThreshold} >= #segments
-
-      * **threshold** (_optional_, default: `0.85`)<br/>
-        consolidate `IFF {threshold} > segment_bytes / (all_segment_bytes / #segments)`
-
-    * **bytes_accum** (_optional_, for default values use: `{}`)<br/>
-
-      * **segmentThreshold** (_optional_, default: `300`; to disable use: `0`)<br/>
-        apply consolidation policy IFF {segmentThreshold} >= #segments
-
-      * **threshold** (_optional_, default: `0.85`)<br/>
-        consolidate `IFF {threshold} > (segment_bytes + sum_of_merge_candidate_segment_bytes) / all_segment_bytes`
-
-    * **count** (_optional_, for default values use: `{}`)
-
-      * **segmentThreshold** (_optional_, default: `300`; to disable use: `0`)<br/>
-        apply consolidation policy IFF {segmentThreshold} >= #segments
-
-      * **threshold** (_optional_, default: `0.85`)<br/>
-        consolidate `IFF {threshold} > segment_docs{valid} / (all_segment_docs{valid} / #segments)`
-
-    * fill: (optional)
-      if specified, use empty object for default values, i.e. `{}`
-
-      * **segmentThreshold** (_optional_, default: `300`; to disable use: `0`)<br/>
-        apply consolidation policy IFF {segmentThreshold} >= #segments
-
-      * **threshold** (_optional_, default: `0.85`)<br/>
-        consolidate `IFF {threshold} > #segment_docs{valid} / (#segment_docs{valid} + #segment_docs{removed})`
+* **cleanupIntervalStep** (_optional_; default: `10`; to disable use: `0`)<br/>
+  wait at least this many commits between removing unused files in the
+  ArangoSearch data directory
+  for the case where the consolidation policies merge segments often (i.e. a
+  lot of commit+consolidate). A lower value will cause a lot of disk space to
+  be wasted
+  for the case where the consolidation policies rarely merge segments (i.e.
+  few inserts/deletes). A higher value will impact performance without any
+  added benefits.
+  >With every "commit" or "consolidate" operation a new state of the view
+   internal data-structures is created on disk
+   old states/snapshots are released once there are no longer any users
+   remaining
+   however, the files for the released states/snapshots are left on disk, and
+   only removed by "cleanup" operation.
```

Comment:
Please check if this is rendered correctly as blockquote, Gitbook's Markdown parser is a bit strange some times